vector-search-overview.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Vector Search Overview
+summary: Learn about Vector Search in TiDB Cloud. This feature provides an advanced search solution for performing semantic similarity searches across various data types, including documents, images, audio, and video.
+---
+
+# Vector Search Overview
+
+TiDB Vector Search provides an advanced search solution for performing semantic similarity searches across various data types, including documents, images, audio, and video. This feature enables developers to easily build scalable applications with generative artificial intelligence (AI) capabilities using familiar MySQL skills.
+
+<CustomContent platform="tidb">
+
+> **Warning:**
+>
+> The vector search feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

Comment:
```suggestion
> The vector search feature is experimental and may be not fully functional. It is not recommended that you use it in the production environment. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

Remove "This feature might be changed or removed without prior notice." as this feature is experimental because of stability issues, not production decision issues.

Reply:
Updated via https://github.com/pingcap/docs/pull/18779/commits/a301f355322c72dcf8ca7453e4e0841b0a142e0e.